### PR TITLE
Display the properties for array type

### PR DESCRIPTION
**Pull request by shatakshiLonkar**

### CODE REVIEW

File: src/browser/modules/EditorInfo/utils.js [hunk 1]
```diff
@@ -34,7 +34,14 @@ export const getStringValue = value => {
     return value ? 'true' : 'false'
   }
 
-  // FIXME handle if value is an array
+  if (typeof value === 'object') {
```
Comment (rahul-tarafdar-synerzip) [inline src/browser/modules/EditorInfo/utils.js]: i need to see it in working.
let me know when we can.

Comment (rahul-tarafdar-synerzip) [reply src/browser/modules/EditorInfo/utils.js]: use Array.isArray(value) to check if value is an array or not

File: src/browser/modules/EditorInfo/utils.js [hunk 2]
```diff
@@ -34,7 +34,14 @@ export const getStringValue = value => {
...
+    let propertyArray = []
+    for (let i in value) {
+      propertyArray.push(value[i], ', ')
+    }
+    propertyArray.pop()
+    return propertyArray
```
Comment (rahul-tarafdar-synerzip) [inline src/browser/modules/EditorInfo/utils.js]: can be simplified with 
return value.join(', ');

File: src/browser/modules/EditorInfo/utils.js (hunk 1 above)
Comment (shatakshiLonkar) [reply src/browser/modules/EditorInfo/utils.js]: Its done

File: src/browser/modules/EditorInfo/utils.js (hunk 2 above)
Comment (shatakshiLonkar) [reply src/browser/modules/EditorInfo/utils.js]: Its Done.

**Review by rahul-tarafdar-synerzip — APPROVED** [review]

**Pull request merged**